### PR TITLE
Add pwd in search paths

### PR DESCRIPTION
**Pull request by pbrunet**
Fix # 403

### CODE REVIEW

File: pythran/transformations/handle_import.py [hunk 1]
```diff
@@ -311,18 +311,17 @@ def import_module(self, name):
     def generate_ImportList(self):
         """List of imported functions to be added to the main module.  """
         import_list = []
-        for _, mod in self.modules.items():
+        for mod in self.modules.itervalues():
```
Comment (serge-sans-paille) [inline pythran/transformations/handle_import.py]: ``values()`` for py3 compat?

File: pythran/transformations/handle_import.py [hunk 2]
```diff
@@ -311,18 +311,17 @@ def import_module(self, name):
...
             if mod.is_main_module:
                 # don't need to import anything from the main module
                 continue
-            for alias, module_name in mod.dependent_modules.items():
+            for alias, module_name in mod.dependent_modules.iteritems():
```
Comment (serge-sans-paille) [inline pythran/transformations/handle_import.py]: :-)

**Comment by serge-sans-paille** [comment]
LGTM as soon as Travis agrees

File: scripts/pythran [hunk 3]
```diff
@@ -149,6 +150,9 @@ try:
 
     module_name, ext = os.path.splitext(os.path.basename(args.input_file))
 
+    # Add compiled module path to search for imported modules
+    sys.path.append(os.path.dirname(args.input_file))
+
```
Comment (serge-sans-paille) [inline scripts/pythran]: sorry to comment now that travis is ok, but ``scripts/pythran`` seems a bad place to add this: what if we use pythran as a library ? the path would not be added... (as in ``%%pythran`` magic)

**Comment by pbrunet** [comment]
Fixed.

For %%pythran magic, it would not have been a problem because pwd is already added in sys.path as it is an interactive session.

**Comment by serge-sans-paille** [comment]
On Sun, Mar 22, 2015 at 10:45:22AM -0700, pbrunet wrote:
> Fixed.
> 
> For %%pythran magic, it would not have been a problem because pwd is already added in sys.path as it is an interactive session.

Correct! Thanks for the fix!

**Pull request merged**